### PR TITLE
[docm] Fix instance code

### PR DESCRIPTION
**Pull request by webvs2**
Closes #<issue number>

### CODE REVIEW

**Comment by jonirrings** [comment]
I will check it later, thx

**Comment by webvs2** [comment]
There was an error when I ran the example yesterday, and the method of entry has changed after understanding it

**Comment by jonirrings** [comment]
great, but `{schema:schema}` may be shorten as `{schema}`

File: src/content/code/code.md [hunk 1]
```diff
@@ -304,7 +304,7 @@ var schema = buildSchema(`
 
 var root = { hello: () => 'Hello world!' };
 
-graphql(schema, '{ hello }', root).then((response) => {
+graphql({schema:schema, source:'{ hello }', rootValue:root}).then((response) => {
```
Comment (jonirrings) [inline src/content/code/code.md]: `schema:schema` -> `schema`

File: src/content/graphql-js/Tutorial-GettingStarted.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ var root = {
 };
 
 // 运行 GraphQL query '{ hello }' ，输出响应
-graphql(schema, '{ hello }', root).then((response) => {
+graphql({schema:schema, source:'{ hello }', rootValue:root}).then((response) => {
```
Comment (jonirrings) [inline src/content/graphql-js/Tutorial-GettingStarted.md]: same as above

**Review by jonirrings — CHANGES_REQUESTED** [review]
just a few to fix

File: src/content/code/code.md (hunk 1 above)
Comment (webvs2) [reply src/content/code/code.md]: Ok, I'll adjust

**Review by jonirrings — APPROVED** [review]
Great, thanks for your contribution.

**Pull request merged**